project-2/Makefile

```diff
@@ -1,7 +1,8 @@
 CC = g++
 
-edit : main.o linked.o hpfnp.o
-	g++ -o test main.o linked.o hpfnp.o
+edit : main.o linked.o hpfnp.o roundRobin.o FCFS.o stats.o
+	g++ -o test main.o linked.o hpfnp.o roundRobin.o FCFS.o \
```

Comment:
sorry I forgot to ask earlier. Could you add the following s.t. g++ will compile in C++11 standard: 

g++ -std=c++11 ...

Thank you, Travis. 

Quan